integration/test_tor.py

```diff
@@ -33,8 +33,8 @@
 def test_onion_service_storage(reactor, request, temp_dir, flog_gatherer, tor_network, tor_introducer_furl):
     carol = yield _create_anonymous_node(reactor, 'carol', 8008, request, temp_dir, flog_gatherer, tor_network, tor_introducer_furl)
     dave = yield _create_anonymous_node(reactor, 'dave', 8009, request, temp_dir, flog_gatherer, tor_network, tor_introducer_furl)
-    yield util.await_client_ready(carol, minimum_number_of_servers=2, timeout=60)
-    yield util.await_client_ready(dave, minimum_number_of_servers=2, timeout=60)
+    yield util.await_client_ready(carol, minimum_number_of_servers=2, timeout=600)
+    yield util.await_client_ready(dave, minimum_number_of_servers=2, timeout=600)
```

Comment:
I guess it might be nice to centralize this timeout definition somewhere.

Reply:
Actually, some of these might be able to go away ... I think I (or someone) put them all up to 600 because "sometimes" key-generation ("or whatver") in Chutney can take a long time, but I didn't try to put them back down lower I guess.